test/spec/modules/admixerBidAdapter_spec.js

```diff
@@ -0,0 +1,122 @@
+import {expect} from 'chai';
+import {spec} from 'modules/admixerBidAdapter';
+import {newBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'admixer';
+const ENDPOINT_URL = '//inv-nets.admixer.net/prebid.1.0.aspx';
```

Comment:
This URL should be updated to https to match the corresponding request in the bidAdapter.js file.